tests/suites/storage/charm_storage.sh

```diff
@@ -0,0 +1,146 @@
+run_charm_storage() {
+	echo
+
+	model_name="test-charm-storage"
+	file="${TEST_DIR}/${model_name}.log"
+
+	ensure "${model_name}" "${file}"
+
+	echo "Assess create-storage-pool"
+	juju create-storage-pool -m "${model_name}" loopy loop size=1G
+	juju create-storage-pool -m "${model_name}" rooty rootfs size=1G
+	juju create-storage-pool -m "${model_name}" tempy tmpfs size=1G
+	juju create-storage-pool -m "${model_name}" ebsy ebs size=1G
+	echo "create-storage-pool PASSED"
+
+	# Assess the above created storage pools.
+	echo "Assessing storage pool"
+	if [ "${BOOTSTRAP_PROVIDER:-}" == "ec2" ]; then
+		juju list-storage-pools -m "${model_name}" --format json | jq '.ebs | .provider' | check "ebs"
+		juju list-storage-pools -m "${model_name}" --format json | jq '.["ebs-ssd"] | .provider' | check "ebs"
+		juju list-storage-pools -m "${model_name}" --format json | jq '.tmpfs | .provider' | check "tmpfs"
+		juju list-storage-pools -m "${model_name}" --format json | jq '.loop | .provider' | check "loop"
+		juju list-storage-pools -m "${model_name}" --format json | jq '.rootfs | .provider' | check "rootfs"
+	else
+		juju list-storage-pools -m "${model_name}" --format json | jq '.rooty | .provider' | check "rootfs"
+		juju list-storage-pools -m "${model_name}" --format json | jq '.tempy | .provider' | check "tmpfs"
+		juju list-storage-pools -m "${model_name}" --format json | jq '.loopy | .provider' | check "loop"
+		juju list-storage-pools -m "${model_name}" --format json | jq '.ebsy | .provider' | check "ebs"
+	fi
+	echo "Storage pool PASSED"
+
+	# Assess charm storage with the filesystem storage provider
+	echo "Assessing filesystem rootfs"
+	juju deploy ./testcharms/charms/dummy-storage-fs --series="jammy" --storage data=rootfs,1G
```

Comment:
I don't think so, check internal implementation of the python test and you see each pool is  being assigned a default storage capacity of 1G.https://github.com/juju/juju/blob/0521fd12a4712268f8e4ed4b7970944d7130e559/acceptancetests/assess_storage.py#L249

Reply:
I mean Juju (for the most part) probably ignores it. Since rootfs storage is just a dir on the machine's file system, the storage size of a rootfs drive I think is just set to the capacity of the instance's storage drive, hence the 7.6Gb resulting storage unit